bitnami/sonarqube/README.md

```diff
@@ -101,6 +101,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `maxHeapSize`          | Maximum heap size for SonarQube                                                    | `2048m`            |
 | `startTimeout`         | Timeout for the application to start in seconds                                    | `150`              |
 | `extraProperties`      | List of extra properties to be set in the sonar.properties file (key=value format) | `[]`               |
+| `LDAPGroupRequest`     | Property ldap.group.request                                                        | `[]`               |
```

Comment:
I'm worried about adding this value not following the standard. If more LDAP related values are added, we will need to release a new major in the future in order to rename it.

We already have support for LDAP in other charts, see `mariadb-galera`,`postgresql`, `airflow`, `grafana` or `rabbitmq`, and @fmulero recently adapted all of them to have a standard structure.

To follow the standard, LDAP-related values should be grouped under `ldap.*`, and should have additional values, such as `ldap.enabled`, `ldap.uri`, etc.

Sonarqube can be configured using env variables, so we would just need to map the `ldap.*` values to the env variables described at https://docs.sonarqube.org/latest/setup/environment-variables/

Alternatively, we could release a new version of the `bitnami/sonarqube` image setting the LDAP configuration in the properties file.